src/image/ImageAnnotator.ts

```diff
@@ -112,8 +116,16 @@ export default class ImageAnnotator extends BaseAnnotator {
             return;
         }
 
+        let shape = null;
         if (isRegion(annotation)) {
-            const transformedShape = getTransformedShape(annotation.target.shape, rotation);
+            // eslint-disable-next-line prefer-destructuring
+            shape = annotation.target.shape;
+        } else if (isDrawing(annotation)) {
+            shape = getShape(annotation.target.path_groups);
+        }
+
+        if (shape) {
+            const transformedShape = getTransformedShape(shape, rotation);
 
             scrollToLocation(this.annotatedEl, referenceEl, {
                 offsets: centerRegion(transformedShape),
```

Comment:
It doesn't really seem ideal to have region-specific logic affecting drawing annotations. Should we be using `centerDrawing`, isntead?